### PR TITLE
Prevent reordering of home screen buttons due to text resizing

### PR DESCRIPTION
**Pull request by phillipm**
Note: doesn't prevent home screen buttons from staggering in a different order when in demo mode, but does prevent button reordering when the home screen is paused and resumed.

Follow up to https://github.com/dimagi/commcare-odk/pull/1040

Issue described in http://manage.dimagi.com/default.asp?215785

### CODE REVIEW

**Comment by wpride** [comment]
from #1040 :

refreshview

@phillipm here's what I'm seeing just implementing that change - is this the desired/acceptable output? Buttons still misaligned, logout button on right side cc @ctsims @amstone326 @biyeun

**Comment by biyeun** [comment]
yeah, those buttons should have consistent groups of height...that looks jarring

**Comment by biyeun** [comment]
e.g. the vibrant colors should all be the same hieght

File: app/src/org/commcare/android/view/ResizingTextView.java [hunk 1]
```diff
@@ -24,27 +23,29 @@
...
+
         setParams(context, attrs);
     }
 
     @Override
     protected void onTextChanged(CharSequence text, int start, int lengthBefore, int lengthAfter) {
-        this.reset();
+        if (!getText().equals(text)) {
+            resetOriginalTextSize();
+        }
```
Comment (phillipm) [inline app/src/org/commcare/android/view/ResizingTextView.java]: The fix

**Comment by phillipm** [comment]
screen

**Pull request merged**